pages.de/common/pip-install.md

```diff
@@ -0,0 +1,24 @@
+# pip install
+
+> Installiere Python-Pakete.
+> Weitere Informationen: <https://pip.pypa.io>.
+
+- Installiere ein Paket:
+
+`pip install {{paketname}}`
+
+- Installiere eine spezifische Paketversion:
+
+`pip install {{paketname}}=={{paketversion}}`
```

Comment:
```suggestion
`pip install {{paket_name}}=={{paket_version}}`
```

Reply:
hm. In german this is one word, that's why I chose to make it one word here. Do you still want me to split it?

Reply:
Ok, I don't speak German. @marchersimon, could you help?

Reply:
Yep, it's indeed just one word, so we sholud leave it like it is.